packages/client/src/pages/EmissionsMetricsPage/CarbonIntensityMap/CarbonIntensityMap.tsx

```diff
@@ -25,6 +25,7 @@ const CarbonIntensityMap = (): ReactElement => {
     AWS: AWSMap,
     GCP: GCPMap,
     Azure: AzureMap,
+    AliCloud: AWSMap,
```

Comment:
This needs to be mapped to an AliCloud map. But I will provide it as promised :)